UnityProject/Assets/Scripts/Botany/hydroponics tray/HydroponicsTray.cs

```diff
@@ -237,7 +226,7 @@ public override void UpdateMe()
 		{
 			if (weedLevel < 10)
 			{
-				weedLevel = weedLevel + ((0.1f) * (0.1f));
+				weedLevel += 0.01f;
```

Comment:
there supposed to be weed growth rate here,
WeedLevel += .1f * (plantData.WeedGrowthRate /100f)
Whatever it is

Reply:
I tried leaving the logic the same way as it was before.

Reply:
Need to add an issue for it, so someone/me Rounders to add it in again

Reply:
*Remembers